src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindBadGenomicKmersSparkUnitTest.java

```diff
@@ -0,0 +1,95 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceWindowFunctions;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Unit tests for FindBadGenomicKmersSpark
+ */
+public class FindBadGenomicKmersSparkUnitTest extends BaseTest {
+
+    private static final int KMER_SIZE = SVConstants.KMER_SIZE;
+    private static final String REFERENCE_FILE_NAME = hg19MiniReference;
+
+    @Test(groups = "spark")
+    public void badKmersTest() throws IOException {
+        final File refSequenceFile = File.createTempFile("genomic_sequence", "txt");
+        refSequenceFile.deleteOnExit();
```

Comment:
The `IOUtils` method is an imperfect solution for an imperfect world :) In its defense, back when GATK test authors were responsible for manual cleanup of their temp files we had terrible problems with stray files getting left in temp directories on each run (particularly indices, which are easy to forget), and these problems were mostly resolved by telling comp bios "just always call `BaseTest.createTempFile()`"